symengine/python/symengine/lib/symengine_wrapper.pxd

```diff
@@ -0,0 +1,68 @@
+cimport symengine
+from symengine cimport RCP
+
+cdef class Basic(object):
+    cdef RCP[const symengine.Basic] thisptr
```

Comment:
Python classes are declared here and then implemented in the pyx file. This is so that these classes can be cimported by other modules.